### DISCUSSION TITLE
Blocking rotation

### DISCUSSION THREAD

**Discussion by Sebi2405** [stephanrauh/ngx-extended-pdf-viewer#1219] [Q&A] [answered]
Hi, 
Is it possible to block document rotation by using Shift+L/R?
My implementation doesn't need rotation and I'd like to disable it.
Thanks!

**Comment by stephanrauh** [comment, marked as answer]
Adding the attribute `[ignoreKeys]="['L', 'R']"` should do the trick. Read the full story here: https://pdfviewer.net/extended-pdf-viewer/keyboard

**Reply by Sebi2405** [reply]
Thanks, it works.
I'm having another problem at Print preview. For some files, it shows at the bottom of every page the URL where the application is deployed. Have you encountered this before?

**Reply by stephanrauh** [reply]
No, I haven't. Maybe it's your printer driver. Does it add the URL when you print an arbitrary page of the browser?

Basically, the print algorithm simply draws the PDF file on canvases and prints them.

**Reply by Sebi2405** [reply]
Indeed, it's the default behavior of the printer driver. Thanks!
